projects/packages/plans/src/class-current-plan.php

```diff
@@ -390,4 +399,35 @@ function_exists( 'wpcom_feature_exists' ) &&
 
 		return false;
 	}
+
+	/**
+	 * Retrieve site-specific features for Simple sites.
+	 *
+	 * We're caching the data for the lifetime of the request, because it can be slow to calculate,
+	 * and it can be called multiple times per single request.
+	 *
+	 * We intentionally don't use object caching or any other type of persistent caching,
+	 * in order to avoid complex cache invalidation on subscription addition or removal.
+	 *
+	 * @see Jetpack_Gutenberg::get_site_specific_features()
+	 *
+	 * @return array
+	 */
+	public static function get_simple_site_specific_features() {
```

Comment:
This method is copied as is from [here](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/jetpack/class.jetpack-gutenberg.php#L1147-L1175).